sdk/keyvault/keyvault-keys/swagger/README.md

```diff
@@ -59,3 +59,33 @@ directive:
     transform: >
       $["x-ms-client-name"] = "encodedPolicy";
 ```
+
+### Remove OKP and OKP-HSM key types
```

Comment:
These are required since we are directly re-exporting the generated types for these enums. Once we have a Swagger without OKP we can back these out.

Reply:
You can use https://github.com/Azure/azure-rest-api-specs/blob/release-keyvault-Microsoft.KeyVault-7.4/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/keys.json for now. It'll be merged early next week.